salt/metalk8s/kubernetes/cni/calico/deployed.sls

```diff
@@ -4389,12 +4477,38 @@ spec:
               name: cni-net-dir
           securityContext:
             privileged: true
+        # This init container mounts the necessary filesystems needed by the BPF data plane
+        # i.e. bpf at /sys/fs/bpf and cgroup2 at /run/calico/cgroup. Calico-node initialisation is executed
+        # in best effort fashion, i.e. no failure for errors, to not disrupt pod creation in iptable mode.
```

Comment:
Wondering if we really should include it, but I guess it's not going to cause issues (just a slightly longer startup time)...